src/components/UI/TextBlock.vue

```diff
@@ -9,17 +9,25 @@ const props = defineProps({
         type: String,
         default: 'Text Content',
     },
+    labelClass: {
```

Comment:
Set these default styles up inside this file, trying to keep all the styles in here. Once it get too big we'll probably split it up and import those files.

https://github.com/Humphreyj/Template/blob/main/src/CSS/componentStyles.js#L3